routes/api.php

```diff
@@ -1070,18 +1070,20 @@
                 ]
             )->name('api.users.restore');
 
-        }); 
+        });
+
+    Route::match(['put', 'patch'], '{user}/update', [Api\UsersController::class, 'update'])
```

Comment:
This is causing a method not found error. https://github.com/snipe/snipe-it/issues/15435

Reply:
Fixed via https://github.com/snipe/snipe-it/pull/15440